clients/java/src/main/java/io/unitycatalog/client/auth/OAuthTokenProvider.java

```diff
@@ -28,24 +28,16 @@ class OAuthTokenProvider implements TokenProvider {
   private static final long DEFAULT_LEAD_RENEWAL_TIME_SECONDS = 30L;
```

Comment:
for all the package private classes, can you add in the docs that this is a internal class, not to be used directly.